tests/dragonfly/tiering_test.py

```diff
@@ -1,39 +1,81 @@
-from . import dfly_args
-
 import async_timeout
 import asyncio
+import itertools
+import pytest
+import random
 import redis.asyncio as aioredis
 
-BASIC_ARGS = {"port": 6379, "proactor_threads": 1, "tiered_prefix": "/tmp/tiering_test_backing"}
+from . import dfly_args
+from .seeder import StaticSeeder
+
+
+BASIC_ARGS = {"port": 6379, "proactor_threads": 4, "tiered_prefix": "/tmp/tiering_test_backing"}
 
 
-# remove once proudct requirments are tested
+@pytest.mark.opt_only
 @dfly_args(BASIC_ARGS)
-async def test_tiering_simple(async_client: aioredis.Redis):
-    fill_script = """#!lua flags=disable-atomicity
-        for i = 1, 100 do
-            redis.call('SET', 'k' .. i, string.rep('a', 3000))
-        end
+async def test_basic_memory_usage(async_client: aioredis.Redis):
     """
+    Loading 1GB of mixed size strings (256b-16kb) will keep most of them on disk and thus RAM remains almost unused
+    """
+
+    seeder = StaticSeeder(
+        key_target=200_000, data_size=2048, variance=8, samples=100, types=["STRING"]
```

Comment:
Is the comment above correct?

Reply:
Yes, the table takes up a lot of space, real memory usage is 1gb